Diegogagan2587/Book-a-concert-front-end
# Feat/testing implementation

# Implemented Changes: 
**NOTE** This Changes continue on the next pull request:
https://github.com/Diegogagan2587/Book-a-concert-front-end/pull/67
## Set up Jest as a testing library.

- Install the Jest library using the next commands:
  - `npm install --save-dev jest` 
  - `npm install --save-dev babel-jest @babel/core @babel/preset-env`
  - `npm install @testing-library/react --save-dev`
- Create the `babel.config.cjs` file to set up jest + react + vite: 
```
module.exports = {
    presets: ['@babel/preset-env', '@babel/preset-react'],
  };
```
- Add a `src/__tests__/sum.test.js` sample test for verifying Jest is working properly.
- in .eslintrc.cjs add `env: { jest:true }` To avoid no-undef error.

**Note** We did not install next packagaes in this pull requeste because they are already installed:
- `@babel/preset-react`
